### PR TITLE
Setup prettier

### PR DESCRIPTION
**Pull request by arturovt**
Hey @nateplusplus , great project! No functional changes in this PR, just installs the prettier and runs it when the `git commit` is executed.

### CODE REVIEW

**Comment by nateplusplus** [comment]
@arturovt  this is a great idea!

What do you think about also integrating it with the existing eslint config? I was reading this article about it: https://www.robinwieruch.de/prettier-eslint/

Seems like it just requires a couple more package installs and some edits to the `.eslitrc`.

File: src/pushin.js [hunk 1]
```diff
@@ -1,10 +1,90 @@
...
+
+  document.body.appendChild(scrollCounter);
+
+  window.addEventListener('scroll', () => {
+    debuggerContent.innerText = `Scroll position: ${Math.round(
+      window.pageYOffset
+    )}px`;
+  });
+}
+
```
Comment (nateplusplus) [inline src/pushin.js:80]: @arturovt it looks like a lot of extra code is being introduced here. These functions should be part of the PushIn class. Any idea what happened here?

File: src/pushin.js [hunk 2]
```diff
@@ -114,13 +194,9 @@ class pushIn {
     const { top } = this.scene.getBoundingClientRect();
 
     let inpoints = [top];
-    if (elem.dataset.hasOwnProperty('pushinFrom')) {
+    if (hasOwnProperty.call(elem.dataset, 'pushinFrom')) {
       inpoints = elem.dataset.pushinFrom.split(',');
-      inpoints = inpoints.map(inpoint => parseInt(inpoint.trim()));
-    } else if (i === 0 && this.scene.dataset.hasOwnProperty('pushinFrom')) {
```
Comment (nateplusplus) [inline src/pushin.js:122]: @arturovt  here is another example of a method being removed for some reason.

**Review by nateplusplus — CHANGES_REQUESTED** [review]
@arturovt I saw your new commit, and it seems like some additional changes were made that may have been unintentional. I've noted them in my review here.

File: src/pushin.js (hunk 2 above)
Comment (arturovt) [reply src/pushin.js:122]: Yeah, I didn't think it's used, but I saw you fixed it another PR. Lint told me that it's an unused variable.

File: src/pushin.js (hunk 1 above)
Comment (arturovt) [reply src/pushin.js]: Reverted it back, the lint was telling that methods should be functions because they're not using `this`.

Comment (nateplusplus) [reply src/pushin.js]: Oh interesting, that is a good point.
Thanks for reverting it back for now. I might look into how to refactor that at some point.

File: src/pushin.js (hunk 2 above)
Comment (nateplusplus) [reply src/pushin.js:122]: Got it, yeah it will be fixed in the next patch release. Thanks!

**Comment by nateplusplus** [comment]
@arturovt - thanks again for your help!

There are some issues with some of the imports in the unit tests still using camel case after it was changed to pascal case, but I can easily fix that before merging to main.

I am merging this into the next patch release now, which will probably go out tonight or tomorrow.

**Pull request merged**